### ISSUE TITLE
HPB/High Performance Backend Talk Participants Limited to 7-8 by Turnserver Quota

### ISSUE THREAD

**Issue by a-blind-programmer** [nextcloud/all-in-one#1121]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

### Steps to reproduce
1. Start Call with Video
2. Have 7 More Participants Join Call with Video
3.

### Expected behavior 
All participants should be in the same call with video.  This is the same issue as #400 

### Actual behavior 
Every participant that joins after the 7th participant gets an "error while establishing connection".  This is because of the line "total-quota=100" in /etc/turnserver.conf.  Changing this to "total-quota=0" appears to fix the issue.  Each participant behind a NAT makes a TURN connection for every other participant.  100 connections to the TURN server can easily be surpassed with less than 10 participants.

### Host OS 
Debian 11

#### Nextcloud AIO version 
1.7.1

**Comment by MartinM01** [comment]
We tested it with 12 people in one room and the result was fine.
We will test it further.

**Comment by a-blind-programmer** [comment]
I updated the issue to detail that the participants should be behind a NAT.  Were all 12 people in the test behind a NAT?

**Comment by szaimen** [comment]
That is interesting! Can you open a PR with that change? Thanks!

**Comment by MartinM01** [comment]
This Nexcloud instance is a cloud server with public IP.
The users should all be at home or in the company behind a router with NAT.

Is that what you mean?

**Comment by szaimen** [comment]
No, apply your change in the file in this repo and submit a PR!

**Comment by szaimen** [comment]
This is now released with v2.0.2 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel)

**Comment by MartinM01** [comment]
We have tested it with beta v2.0.2: ~18 participants, all video active and 1x screen sharing. 
It ran surprisingly well. We did not reach the limit.

cloud server resources:
4 cores
8gb RAM
